### PR TITLE
Send CI env variable to conda-build

### PR DESCRIPTION
**Pull request by isuruf**
Checklist
* [ ] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by isuruf** [comment]
Ready for review

File: recipe/run_conda_forge_build_setup_osx [hunk 1]
```diff
@@ -39,6 +39,13 @@ if [ ! -z "$CONFIG" ]; then
     echo "CONDA_BUILD_SYSROOT:" >> ./.ci_support/${CONFIG}.yaml
     echo "- ${CONDA_BUILD_SYSROOT}" >> ./.ci_support/${CONFIG}.yaml
     echo "" >> ./.ci_support/${CONFIG}.yaml
+
+    if [ ! -z "$CI" ]; then
+        echo "CI:" >> ./.ci_support/${CONFIG}.yaml
```
Comment (jakirkham) [inline recipe/run_conda_forge_build_setup_osx]: Maybe add a blank line before? Same for the rest as well.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {% set name = "conda-forge-ci-setup" %}
-{% set version = "2.2.1" %}
+{% set version = "2.2.2" %}
```
Comment (jakirkham) [inline recipe/meta.yaml]: Would do 2.3.0 as it is a new feature.

**Comment by jakirkham** [comment]
Looks good. Though I guess we need to wait for PR ( https://github.com/conda-forge/conda-smithy/pull/1068 ).

File: recipe/run_conda_forge_build_setup_osx (hunk 1 above)
Comment (isuruf) [reply recipe/run_conda_forge_build_setup_osx]: There's a blank line added in Line 41

**Pull request merged**